### DISCUSSION TITLE
BUG: commit links in PRs do not allow me to review the PR

### DISCUSSION THREAD

**Discussion by dzenanz** [community/community#5316] [Pull Requests]
A detailed description is available [here](https://github.com/sindresorhus/refined-github/issues/4706), as I originally though it was a bug in `Refined-GitHub` extension. We concluded it is a bug in GitHub itself.

**Comment by fregante** [comment]
More specifically, this is an error due to AJAX loading. The commit links in the conversation tab point to the plain "commit" pager rather than the "PR commit" page

**Comment by andersk** [comment]
I’m seeing the commit links probabilistically flip between associated with the PR and disassociated from the PR on page load. If I Shift+Reload the page, I randomly get one or the other.

![Screenshot from 2021-09-02 15-38-48](https://user-images.githubusercontent.com/26471/131925141-82dda1b9-87b5-467d-aed6-fa93ea4c4842.png)
Associated: `https://github.com/zulip/zulip/pull/19568/commits/dea4187affd2d1c6a64dd816b107e2f86e208573`

---

![Screenshot from 2021-09-02 15-38-24](https://user-images.githubusercontent.com/26471/131925137-1afc804d-fbc8-4523-bf41-433f8633499f.png)
Disassociated: `https://github.com/eeshangarg/zulip/commit/dea4187affd2d1c6a64dd816b107e2f86e208573`

**Comment by lsgunth** [comment]
I was seeing this too. It's very annoying when reviewing PRs. However, it seems today it got worse and they are now always going to the disassociated link.
